README.md

```diff
@@ -98,6 +98,15 @@ Hurray! You successfully made a contribution!
 
 ---
 
+## <ins> What if I have a Conflict? </ins>
```

Comment:
```suggestion
## <ins>What if I have a Conflict?<ins>
```

Reply:
Hey, I don't think this would work as these are tags.